extensions/cornerstone/src/toolbarModule.js

```diff
@@ -98,6 +98,15 @@ const definitions = [
     commandName: 'setToolActive',
     commandOptions: { toolName: 'Angle' },
   },
+  {
```

Comment:
Multiple projects are using the master branch of the OHIF and don't need the Cobb Angle, so I recommend taking it out of the toolbar, or maybe put it in `More`

Reply:
good idea!